### PR TITLE
Add ENV var to control autoKiller for seperate process on the HTML validator

### LINKED ISSUE #591: Add ENV var to control autoKillerTimeout for seperate process on the HTML validator
Add an ENV variable to control the auto killer on the separate process for the HTML validator. This is useful when you don't want your validator to be killed after a timeout and want this setting to persist among multiple projects

### PR DESCRIPTION
**Pull request by lyomr**
This PR 
- Creates an environment variable called ROOSEVELT_AUTOKILLER
- ROOSEVELT_AUTOKILLER should be set to `on` or `off` to change autokiller param
- Updates ReadMe to include the environment variable
- Adds unit test in the envParams
- Closes #591 
- Updates CHANGELOG

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -201,6 +201,9 @@ The following is a list of [environment variables](https://en.wikipedia.org/wiki
 - `ROOSEVELT_VALIDATOR`:
   - Set to `detached` to force the HTML validator to run as a detached background process.
   - Set to `attached` to force the HTML validator to run as an attached process.
+- `ROOSEVELT_AUTOKILLER`:
+  - Set to `on` to turn on the HTML validator autokiller on the separate process.
+  - Set to `off`to turn off the HTML validator autokiller on the separe process.
```
Comment (kethinov) [inline README.md]:
```suggestion
  - Set to `off`to turn off the HTML validator autokiller on the separate process.
```

File: README.md [hunk 2]
```diff
@@ -201,6 +201,9 @@ The following is a list of [environment variables](https://en.wikipedia.org/wiki
 - `ROOSEVELT_VALIDATOR`:
   - Set to `detached` to force the HTML validator to run as a detached background process.
   - Set to `attached` to force the HTML validator to run as an attached process.
+- `ROOSEVELT_AUTOKILLER`:
+  - Set to `on` to turn on the HTML validator autokiller on the separate process.
```
Comment (kethinov) [inline README.md]:
```suggestion
  - Set to `on` to spawn a process to kill the HTML validator if it is running in the background and idle for more than a certain amount of time. The timeout can be configured in [app behavior params](https://github.com/rooseveltframework/roosevelt#app-behavior-parameters).
```

File: README.md [hunk 3]
```diff
@@ -201,6 +201,9 @@ The following is a list of [environment variables](https://en.wikipedia.org/wiki
...
+  - Set to `off`to turn off the HTML validator autokiller on the separate process.
```
Comment (kethinov) [inline README.md]:
```suggestion
  - Set to `off`to disable the HTML validator autokiller.
```

**Comment by alallier** [comment]
Are we not doing any command line flags for this, just the ENV?

**Comment by alallier** [comment]
FWIW it's worth in general it seems the Autokill tests don't work in Windows

**Comment by kethinov** [comment]
The tests do work in Windows, but not reliably due to timeouts.

As for adding a command line flag, yeah that makes sense.

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,7 +2,7 @@
 
 ## Next version
 
-- Put your changes here.
+- Added autokiller environment variable to enable/disable the HTML validator autokiller
```
Comment (kethinov) [inline CHANGELOG.md]:
```suggestion
- Added autokiller environment variable to enable/disable the HTML validator autokiller.
```

**Pull request merged**